desktop/cmp/rest/impl/RestGridToolbar.js

```diff
@@ -32,7 +32,7 @@ export const restGridToolbar = hoistCmp.factory({
             filler(),
             gridCountLabel({gridModel, unit}),
             storeFilterField({gridModel, includeFields: model.filterFields}),
-            exportButton({gridModel})
+            model.gridModel.enableExport ? exportButton({gridModel}) : undefined
```

Comment:
Our element factories support an `omit` config which makes this pattern cleaner, so this line can be changed to 
```
exportButton({gridModel, omit: !model.gridModel.enableExport})
```